modules/mongodb/charts.go

```diff
@@ -529,6 +571,31 @@ func (m *Mongo) removeDatabasesFromDBStatsCharts(newDatabases []string) {
 	}
 }
 
+// removeReplicaSetMember removes dimensions for not existing
+// replicate set members
+func (m *Mongo) removeReplicaSetMembers(newMembers []string) {
+	diff := sliceDiff(m.replSetMembers, newMembers)
+	for _, name := range diff {
+		dimID := replicationHeartbeatLatencyDimPrefix + name
+		err := m.charts.Get(replicationHeartbeatLatency).MarkDimRemove(dimID, true)
```

Comment:
Is it possible that "m.charts.Get(replicationHeartbeatLatency)" returns nil?

Reply:
yes, that is possible if we fail to add the chart for some reason. A little bit of an edge case because we use constants to add the charts, but I've added nil checks to be 100% sure that the code will never panic and some logging if the chart is nil